configs/peer/config.json

```diff
@@ -14,10 +15,10 @@
     "PEER_ID": null,
     "BLOCK_TIME_MS": 2000,
     "TRUSTED_PEERS": null,
-    "COMMIT_TIME_LIMIT_MS": 4000,
+    "COMMIT_TIME_LIMIT_MS": 40000,
```

Comment:
Why 10x increase...? O_o

Reply:
Maybe we should have a separate config for tests/benches? Like we already have `iroha_test_config.json`.

I think `configs/peer/config.json` should be a reference for real world blockchains, not a range for tests/benches.

Reply:
This was me trying to get things to work when there were bugs. It has been removed now.